frontend/css/report.css

```diff
@@ -0,0 +1,1049 @@
+.report-header{
```

Comment:
Was this file originally written using css? Instead of sass / less?

There are a lot of comments in this css file. Please clean it up and leave only the useful ones.

Reply:
For all color codes, either go all caps or all small.

Reply:
Yes this file was originally written using css
- unnecessary comments are removed now
- all color codes are now changed to caps
- now file includes only #rgba codes